src/main/java/com/coherent/training/selenium/stanila/base/BaseClass.java

```diff
@@ -1,31 +1,34 @@
 package com.coherent.training.selenium.stanila.base;
-
-import com.coherent.training.selenium.stanila.pages.LoginPOM;
+import com.coherent.training.selenium.stanila.pages.*;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.asserts.SoftAssert;
-import java.time.Duration;
-
 public class BaseClass {
```

Comment:
let's rename it to BaseTest, move it to src/test module

Reply:
Implemented